indigo/distribution.yaml

```diff
@@ -12877,6 +12877,22 @@ repositories:
       url: https://github.com/ros-visualization/rqt_moveit.git
       version: master
     status: maintained
+  rqt_mrta:
+    doc:
+      type: git
+      url: https://github.com/adrianohrl/rqt_mrta.git
+      version: 0.2.5
```

Comment:
I'd recommend that this also use a branch instead of a tag so that you can update the docs easily.